spec/namespaces/cat.yaml

```diff
@@ -2126,63 +2130,67 @@ components:
     cat.recovery::query.index:
       name: index
       in: query
-      description: Comma-separated list or wildcard expression of index names to limit the returned information.
+      description: |-
+        A comma-separated list of data streams, indexes, and aliases used to limit the request.
+        Supports wildcards (`*`). To target all data streams and indexes, omit this parameter or use `*` or `_all`.
       style: form
       schema:
         type: array
         items:
           type: string
-        description: Comma-separated list or wildcard expression of index names to limit the returned information.
+        description: |-
+          A comma-separated list of data streams, indexes, and aliases used to limit the request.
+          Supports wildcards (`*`). To target all data streams and indexes, omit this parameter or use `*` or `_all`.
       explode: true
     cat.recovery::query.s:
       name: s
       in: query
-      description: Comma-separated list of column names or column aliases to sort by.
+      description: A comma-separated list of column names or column aliases to sort by.
       style: form
       schema:
         type: array
         items:
           type: string
-        description: Comma-separated list of column names or column aliases to sort by.
+        description: A comma-separated list of column names or column aliases to sort by.
       explode: true
     cat.recovery::query.time:
       name: time
       in: query
-      description: The unit in which to display time values.
+      description: Specifies the units for time. For example, `5d` or `7h`. For more information, see [Supported units](https://opensearch.org/docs/latest/api-reference/units/).
       schema:
         $ref: '../schemas/_common.yaml#/components/schemas/TimeUnit'
     cat.recovery::query.v:
       name: v
       in: query
-      description: Verbose mode. Display column headers.
+      description: Enables verbose mode, which displays column headers.
       schema:
         type: boolean
         default: false
-        description: Verbose mode. Display column headers.
+        description: Enables verbose mode, which displays column headers.
     cat.repositories::query.cluster_manager_timeout:
       name: cluster_manager_timeout
       in: query
-      description: Operation timeout for connection to cluster-manager node.
+      description: A timeout for connection to the cluster manager node.
```

Comment:
"The" timeout? Or, even better, do we mean "The amount of time allowed to establish a connection to the cluster manager node"?